### ISSUE TITLE
BungeeControl-2.1.1.jar Crashing Bungee (Waterfall)

### ISSUE THREAD

**Issue by FrozenBeard** [kangarko/ChatControl-Pro#1273]
After updating to BungeeControl 2.1.1 (From 2.0.0) I have had 5 Bungee (waterfall) crashes in the last 2 hours, Previous to this I have had 0 Bungee crashes in a year. Status page: http://status.piratemc.com/

The logs show an error related to luckperms, I'm currently using Luckperms 4.4.22, Ive been using LP 4.4.22 since May according to my last modified timecode without any crashes, its only once I updated the BungeeControlBridge yesterday.

This server is running Waterfall version git:Waterfall-Bootstrap:1.14-SNAPSHOT:1f16030:286 by md_5

https://gist.github.com/FrozenBeard/38018f4c35230a593fc87276b4b1ad06

**Comment by kangarko** [comment]
That is some bad luck - I am sorry to hear that!

Now, BungeeControl did not really change anything on 2.1.1 (https://i.imgur.com/tRk2Qll.png), so if that would be the case it would render sooner.

The stacktrace is purely connected to LuckPerms so your best bet is to reach out for that author first. If you find anything else related to MineAcademy plugins (maybe take a look in Spigot servers?) just post below or feel free to create a new issue.

Best
Matej

**Comment by FrozenBeard** [comment]
I removed BungeeControl when I posted this, and It hasn't crashed since.
I went from 2.0.0 to 2.1.1, Im still using the same luckperms version.
It was crashing 5-6 times a day.
I have had to disable all ChatControl cross server bungee chat, since removing it, its been fine.

**Comment by kangarko** [comment]
There will be a major update in cross-server chat tonight, please test using both next BungeeControl and ChatControl. If anything comes I need the following:

- Full ZIP of both BungeeControl and ChatControl folders (make sure to remove MySQL info)
- Startup log from ChatControl server
- Full log from Waterfall until it crashed.

Thank you. I am taking this with full seriousness and will investigate this.
